### PR TITLE
Fix image usage display

### PR DESCRIPTION
**Pull request by majst01**

### CODE REVIEW

File: cmd/tableprinters/image.go [hunk 1]
```diff
@@ -15,10 +15,7 @@ func (t *TablePrinter) ImageTable(data []*models.V1ImageResponse, wide bool) ([]
 		rows [][]string
 	)
 
-	header := []string{"ID", "Name", "Description", "Features", "Expiration", "Status"}
-	if wide {
-		header = []string{"ID", "Name", "Description", "Features", "Expiration", "Status", "UsedBy"}
-	}
+	header := []string{"ID", "Name", "Description", "Features", "Expiration", "Status", "UsedBy"}
 
 	sort.SliceStable(data, func(i, j int) bool { return *data[i].ID < *data[j].ID })
```
Comment (Gerrit91) [inline cmd/tableprinters/image.go:20]: I think this line could even be removed, too.

File: cmd/tableprinters/image.go [hunk 2]
```diff
@@ -38,11 +35,7 @@ func (t *TablePrinter) ImageTable(data []*models.V1ImageResponse, wide bool) ([]
 			usedBy = strings.Join(image.Usedby, "\n")
 		}
 
-		if wide {
-			rows = append(rows, []string{id, name, description, features, expiration, status, usedBy})
-		} else {
-			rows = append(rows, []string{id, name, description, features, expiration, status})
-		}
+		rows = append(rows, []string{id, name, description, features, expiration, status, usedBy})
```
Comment (Gerrit91) [inline cmd/tableprinters/image.go:38]: The old version was showing the UUIDs on `-o wide`: https://github.com/metal-stack/metalctl/blob/v0.11.9/cmd/output/printer.go#L946-L948

**Comment by Gerrit91** [comment]
Thanks for bringing this back. Seems like I missed this one out in the big refactoring. 🙈

File: cmd/tableprinters/image.go (hunk 1 above)
Comment (majst01) [reply cmd/tableprinters/image.go]: done

**Review by Gerrit91 — APPROVED** [review]

**Pull request merged**